library/script-library/src/main/java/uk/gov/gchq/gaffer/script/operation/handler/RunScriptHandler.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.script.operation.handler;
+
+import uk.gov.gchq.gaffer.operation.OperationException;
+import uk.gov.gchq.gaffer.script.operation.RunScript;
+import uk.gov.gchq.gaffer.script.operation.container.Container;
+import uk.gov.gchq.gaffer.script.operation.image.Image;
+import uk.gov.gchq.gaffer.script.operation.platform.ImagePlatform;
+import uk.gov.gchq.gaffer.script.operation.platform.LocalDockerPlatform;
+import uk.gov.gchq.gaffer.script.operation.provider.GitScriptProvider;
+import uk.gov.gchq.gaffer.script.operation.provider.ScriptProvider;
+import uk.gov.gchq.gaffer.store.Context;
+import uk.gov.gchq.gaffer.store.Store;
+import uk.gov.gchq.gaffer.store.operation.handler.OperationHandler;
+
+import java.io.File;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+public class RunScriptHandler implements OperationHandler<RunScript> {
+
+    // comment for json injection
+    private ImagePlatform imagePlatform = new LocalDockerPlatform();
+    private ScriptProvider scriptProvider = new GitScriptProvider();
```

Comment:
I feel these fields should be injected in via constructor/builder, not bound directly to a concrete implementation within the class.